Makefile

```diff
@@ -7,12 +7,12 @@ PACKAGE := github.com/seccomp/containers-golang
 
 sources := $(wildcard *.go)
 
-default.json: $(sources)
+seccomp.json: $(sources)
```

Comment:
I think we have to make this `.PHONY`, right?

Reply:
Good catch! Updated.